tests/binning_test.py

```diff
@@ -586,14 +586,6 @@ def test_binning_low_level_functions_exist():
     binning.make_histogrammed
 
 
-def test_histogram_deprecated_name():
-    da = sc.data.table_xyz(100)
-    x = sc.linspace('x', 0, 1, num=10, unit='m')
-    with pytest.warns(UserWarning):
-        result = sc.histogram(da, bins=x)
-    assert sc.identical(result, da.hist(x=x))
-
-
 def test_bin_deprecated_arguments():
     da = sc.data.table_xyz(100)
     x = sc.linspace('x', 0, 1, num=10, unit='m')
```

Comment:
Isn't this one also being removed? Or does it fail for a different reason now?

Reply:
It was a ruff error, and this deprecation is not removed in this PR. I guess there is no reason to not remove it? This was also deprecated over 2 years ago.